WycliffeAssociates/8woc
# Added username and time to csv export

#### This pull request addresses:
This PR address the issue of not having time and the user in the csv data



#### How to test this pull request:
Try exporting projects and looking at the last two columns, they should have the time and name (if available)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/wycliffeassociates/8woc/347)
